components/Navigation/SectionNavigation.tsx

```diff
@@ -37,7 +37,7 @@ const useStyles = createUseStyles({
 type SectionType = keyof typeof sections;
 type SectionItem = { title: string; href: string };
 
-const SectionNavigation = () => {
+const SectionNavigation = (width, height) => {
```

Comment:
Props are object, so let's break them into constants
```suggestion
const SectionNavigation = ({ width, height }) => {
```